woocommerce/woocommerce-android
# issue/3781-rename-attribute

Closes #3781 - adds the ability to rename a local attribute (renaming global attributes is not supported in our mobile apps).

To test:

* Go to the term list for a local attribute
* Select "Rename" from the options menu
* Enter a new name for the attribute
* Ensure the changes are saved

![attr](https://user-images.githubusercontent.com/3903757/113446065-06642980-93c5-11eb-8606-c3ce743557d5.png)


Update release notes:
